tests/e2e/portal/3D_Anatomical.js

```diff
@@ -25,23 +25,29 @@ async function runTutorial () {
     const studyData = await tutorial.openStudyLink();
 
     const workbenchData = utils.extractWorkbenchData(studyData["data"]);
+    const vtkNodeId = workbenchData["nodeIds"][1];
     await tutorial.waitForServices(
       workbenchData["studyId"],
-      [workbenchData["nodeIds"][1]],
+      [vtkNodeId],
       startTimeout,
       false
     );
 
     await tutorial.waitFor(10000, 'Some time for starting the service');
     await utils.takeScreenshot(page, screenshotPrefix + 'service_started');
 
-    // This study opens in fullscreen mode
-    await tutorial.restoreIFrame();
-
-    const outFiles = [
-      "data.zip"
+    const iframe = await tutorial.getIframe(vtkNodeId);
+    const entitiesListed = [
+      "Vein.vtk",
+      "Artery.vtk",
+      "Bones.e",
```

Comment:
warning: this file's extension looks fisshy: `.e`

Reply:
it's correct